core/const.py

```diff
@@ -1,9 +1,6 @@
 """Module of constants."""
-from collections import namedtuple
 from enum import StrEnum
 
-LookupArgs = namedtuple("fk_lookup_args", ["fk_primary", "related_table", "pk_related"])
-
```

Comment:
This isn't directly part of the FakeDB but was made redundant when we changed the endpoints from querying the FakeDB to SQLite